docs/buildah-build.1.md

```diff
@@ -742,31 +742,12 @@ The `buildah build` command allows building images for all Linux architectures,
 
 **--pull**
 
-When the *--pull* flag is enabled or set explicitly to `true` (with
-*--pull=true*), attempt to pull the latest versions of base and SBOM scanner
-images from the registries listed in registries.conf if a local base or SBOM
-scanner image does not exist or the image in the registry is newer than the one
-in local storage. Raise an error if the base or SBOM scanner image is not in
-any listed registry and is not present locally.
-
-If the flag is disabled (with *--pull=false*), do not pull base and SBOM
-scanner images from registries, use only local versions. Raise an error if a
-base or SBOM scanner image is not present locally.
-
-If the pull flag is set to `always` (with *--pull=always*), pull base and SBOM
-scanner images from the registries listed in registries.conf.  Raise an error
-if a base or SBOM scanner image is not found in the registries, even if an
-image with the same name is present locally.
-
-If the pull flag is set to `missing` (with *--pull=missing*), pull base and
-SBOM scanner images only if they could not be found in the local containers
-storage.  Raise an error if no image could be found and the pull fails.
-
-If the pull flag is set to `never` (with *--pull=never*), do not pull base and
-SBOM scanner images from registries, use only the local versions.  Raise an
-error if the image is not present locally.
-
-Defaults to *true*.
+Pull image policy. The default is **missing**.
+
+- **always**: Always pull the image and throw an error if the pull fails.
+- **missing**: Pull the image only when the image is not in the local containers storage.  Throw an error if no image is found and the pull fails.
+- **never**: Never pull the image but use the one from the local containers storage.  Throw an error if no image is found.
+- **newer**: Pull if the image on the registry is newer than the one in the local containers storage.  An image is considered to be newer when the digests are different.  Comparing the time stamps is prone to errors.  Pull errors are suppressed if a local image was found.
```

Comment:
this doesn't specify what happens with true or false, does it?

Reply:
No we want to hide those, since we do not want people using them any longer.  Just use the newer terminology so it is consistent with Podman.

Reply:
This drops all mention of SBOM scanner images.